rts/lra/lra-client/src/main/java/io/narayana/lra/client/NarayanaLRAClient.java

```diff
@@ -286,25 +286,22 @@ public URI startLRA(URI parentLRA, String clientID, Long timeout, ChronoUnit uni
                     .request()
                     .post();
 
-            // validate the HTTP status code says an LRAInfo resource was created
+            // validate the HTTP status code says an LRA resource was created
             if (isUnexpectedResponseStatus(response, Response.Status.CREATED)) {
                 LRALogger.i18NLogger.error_lraCreationUnexpectedStatus(response.getStatus(), response.toString());
                 throwGenericLRAException(null, INTERNAL_SERVER_ERROR.getStatusCode(),
                         "LRA start returned an unexpected status code: " + response.getStatus());
                 return null;
             }
 
-            // validate that there is an LRAInfo response header holding the LRAInfo id
-            Object lraObject = response.getLastHeader(LRA_HTTP_CONTEXT_HEADER);
+            lra = response.getLocationHeaderAsURI();
```

Comment:
I should have mentioned, this LRAInfo comment is stale too. If you change LRAInfo to LRA, and push a commit up agai,  then merge this PR then my approval would still stand. Otherwise, please let me know if you want me to look again.

Reply:
+1

Reply:
Although there are no longer any references to LRAInfo

Reply:
I see you removed this comment I missed the "-". I will mark this as resolved